3_visualize/src/summarize_site_list.R

```diff
@@ -0,0 +1,46 @@
+summarize_site_list <- function(site_list_path,nwis_daily_data,nwis_inst_data,fileout){
```

Comment:
If we are moving the `sitelist_summary_csv` target to `2_process.R` this file should then move to `2_process/src/`